clusters/build-clusters/build09/openshift-ai/operator.yaml

```diff
@@ -1,24 +1,61 @@
 apiVersion: v1
-kind: Namespace
-metadata:
-  name: openshift-ai-operator
----
-apiVersion: operators.coreos.com/v1
-kind: OperatorGroup
-metadata:
-  name: openshift-ai-operator
-  namespace: openshift-ai-operator
-spec:
-  targetNamespaces:
-  - openshift-ai-operator
----
-apiVersion: operators.coreos.com/v1alpha1
-kind: Subscription
-metadata:
-  name: opendatahub-operator
-  namespace: openshift-ai-operator
-spec:
-  channel: fast
-  name: opendatahub-operator
-  source: community-operators
-  sourceNamespace: openshift-marketplace
+kind: List
+items:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: redhat-ods-operator
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: istio-system
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: knative-serving
+- apiVersion: operators.coreos.com/v1
+  kind: OperatorGroup
+  metadata:
+    name: redhat-ods-operator
+    namespace: redhat-ods-operator
+  spec:
+    targetNamespaces: []
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: rhods-operator
+    namespace: redhat-ods-operator
+  spec:
+    channel: stable
+    name: rhods-operator
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: servicemeshoperator
+    namespace: openshift-operators
+  spec:
+    channel: stable
+    name: servicemeshoperator
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+- apiVersion: datasciencecluster.opendatahub.io/v1
+  kind: DataScienceCluster
+  metadata:
+    name: default-dsc
+  spec:
+    components:
+      dashboard:
+        managementState: Managed
+      kserve:
+        managementState: Managed
+        defaultDeploymentMode: Serverless
+        RawDeploymentServiceConfig: Headed
```

Comment:
@droslean can you apply this yaml and verify if this field is correct?

Reply:
I've opened https://issues.redhat.com/browse/RHAIRFE-471